pages/jobs/index.tsx

```diff
@@ -41,110 +46,228 @@ const CountUp = ({startTime}) => {
 };
 
 export default function Jobs({items, project}) {
+	const [filterOpen, setFilterOpen] = useState(false);
+
+	const router = useRouter();
+
+	const currentRoute = router.asPath;
```

Comment:
Instead of checking the route here, we can grab the query string via next.js' server side context. [See my updates](https://github.com/liferay-frontend/lug-bot-app/pull/18/files#diff-a9709b36f82ab3f4dc11496181bd222a020ae1d13275a83725e5485679acd233R272)